cdap-app-templates/cdap-etl/cdap-etl-core/src/test/java/co/cask/cdap/templates/etl/batch/DefaultAdapterConfigurer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.batch;
+
+import co.cask.cdap.api.schedule.Schedule;
+import co.cask.cdap.api.templates.AdapterConfigurer;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import java.util.Map;
+
+/**
+ * Default implementation of {@link AdapterConfigurer} for ETL Batch Tests.
+ */
+//TODO: Remove/Move this class else where until we figure out how to write tests without AdapterConfigurer dependency
+public class DefaultAdapterConfigurer implements AdapterConfigurer {
```

Comment:
if this is just for tests, it's better to call it MockAdapterConfigurer or something like that.